qiita_db/study.py

```diff
@@ -617,11 +635,14 @@ def investigation(self):
         -------
         Investigation id
         """
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT investigation_id FROM qiita.investigation_study WHERE "
-               "study_id = %s")
-        inv = conn_handler.execute_fetchone(sql, (self._id, ))
-        return inv[0] if inv is not None else inv
+        with TRN:
+            sql = """SELECT investigation_id FROM qiita.investigation_study
```

Comment:
A study can be part of more than one investigation though, right? Hence the `investigation_study` join table.